### PR TITLE
feat: add tenant document

### PR DESCRIPTION
**Pull request by braaar**

### CODE REVIEW

File: schemas/tenant.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    {
+      name: 'name',
+      type: 'string',
+      title: 'Name',
+    },
+    { name: 'id', type: 'string', title: 'ID' },
+    { name: 'email', type: 'string', title: 'Email' },
+    { type: 'string', name: 'type', title: 'Type' },
+    { type: 'string', name: 'environment', title: 'Environment' },
+    {
```
Comment (moripen) [inline schemas/tenant.ts:15]: Maybe have them all on the form
{
   name: '',
   type: '',
   ...
} ? 
Or is this just prettiers doing?

Comment (braaar) [reply schemas/tenant.ts:15]: This was github copilot's doing :)

Comment (moripen) [reply schemas/tenant.ts:15]: The bots are trying to induce OCD in the humans!

Comment (braaar) [reply schemas/tenant.ts]: Done

File: schemas/tenant.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
+export default {
```
[the remark this answers is not in the document]
Comment (braaar) [reply schemas/tenant.ts]: Done.

File: schemas/procore-data.ts
Comment (simenandre) [inline schemas/procore-data.ts:1]: Maybe we should move this to `schemas/objects` to adhere to Sanity typical setups?

File: schemas/tenant.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
+import { defineType } from 'sanity';
+
+export const tenant = defineType({
+  name: 'tenant',
+  type: 'document',
+  title: 'Tenant',
+  groups: [
+    {
```
Comment (simenandre) [inline schemas/tenant.ts:8]:
```suggestion
    { name: 'company', title: 'Company details' },
    {
```

File: schemas/procore-data.ts
Comment (braaar) [reply schemas/procore-data.ts:1]: Could you explain how this is done? I can't seem to figure it out

Comment (braaar) [reply schemas/procore-data.ts:1]: I tried this
```ts
export const schemaTypes = [tenant];

export const objectTypes = [abaxData, procoreData];
```

and this:
```ts
export default defineConfig({
  name: 'default',
  title: 'Branches – Procore / ABAX Backoffice',

  projectId: 'j1f6z3dh',
  dataset: 'production',

  plugins: [deskTool(), visionTool()],

  schema: {
    types: schemaTypes,
    objects: objectTypes,
  },
});

```

But sanity complains that it doesn't know what the abaxData and procoreData objects are.

File: schemas/tenant.ts (hunk 3 above)
Comment (braaar) [reply schemas/tenant.ts:8]: Done.

File: schemas/procore-data.ts
Comment (simenandre) [reply schemas/procore-data.ts:1]: No, you still import it like before. Just place it on a separate directory and import it like before :)

**Pull request merged**